data/json/monsters/mammal.json

```diff
@@ -149,6 +149,32 @@
     "harvest": "mammal_small_fur",
     "flags": [ "SEES", "HEARS", "GOODHEARING", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "HIT_AND_RUN" ]
   },
+  {
+    "id": "mon_cat_kitten",
+    "type": "MONSTER",
+    "copy-from": "mon_cat",
+    "name": "kitten",
+    "description": "A cuddly kitten meowing and scampering amidst the death.  Currently feral.",
+    "default_faction": "cat",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "2000 ml",
+    "weight": "2 kg",
```

Comment:
Kittens should weigh less, I think? Around 1 kilogram, I suppose.

Reply:
Agreed - perhaps reduce the volume to 1000 ml as well? Young kittens are famous for fitting in the palm of your hand.

Reply:
Yeah, volume could be lowered too.